### PR TITLE
Set up .gitignore

### PR DESCRIPTION
**Pull request by erichodges**
Let me know if other items are need for this file. :-)

This is my first OSS PR, lol.

### CODE REVIEW

File: .gitignore
Comment (sagirk) [inline .gitignore]: nit: A space separating the hash from the comment would look 'cleaner'.
```diff
-#General
+# General
```

**Review by sagirk — APPROVED** [review]
👋 Welcome onboard, Eric! Thanks for the PR. 🤗 

A tiny nit below, otherwise LGTM! 🎉

File: .gitignore
Comment (erichodges) [reply .gitignore]: @sagirk Ok cool, I inserted a space and pushed it to my master. Does that fix it, or do I need to do something else?

**Comment by maddhruv** [comment]
added `do-not-merge`, the .gitignore needs to be worked upon

**Comment by erichodges** [comment]
Hi @maddhruv, I'm happy to make changes. :-)

**Comment by JonahMoses** [comment]
@maddhruv It would be helpful if you were able to provide @erichodges with some feedback on what needs to be worked upon before you'd like to see this merged.

Refs: #94

**Comment by maddhruv** [comment]
we are not pretty sure about the framework we will be using for the website and hence not confirmed about the files to be added!
Otherwise general additions are welcome

**Comment by JonahMoses** [comment]
@maddhruv Do you think getting the nit with this PR updated https://github.com/nodejs/website-redesign/pull/96#discussion_r219674025, would suffice enough to then get this PR merged?

Later on when a decision about the framework has been made, additional follow up PRs could cover the necessary additions.

I'm thinking this may help us keep the repo as tidy as we can instead of allowing this PR to remain open for an undetermined amount of time. 

Thoughts?

**Comment by bnb** [comment]
+1 @JonahMoses

File: .gitignore
Comment (sagirk) [reply .gitignore]: @erichodges Could you confirm if the push was successful? Somehow the change is not reflecting here.

**Comment by sagirk** [comment]
+1 @JonahMoses

File: .gitignore
Comment (erichodges) [reply .gitignore]: @sagirk I guess my earlier push (with the space added) was not successful. I just pushed it again and it's showing on the master branch of my fork on github.

**Comment by JonahMoses** [comment]
I don't want to remove @maddhruv's `do-not-merge` until we get buy in from them but LGTM 👍 


Thanks @erichodges for doing this!

**Comment by erichodges** [comment]
@JonahMoses No worries! I want to do more, if possible.

File: .gitignore
Comment (sagirk) [reply .gitignore]: Thanks @erichodges! Looks good now.

**Review by keywordnew — APPROVED** [review]
LGTM.

Congrats on your first PR! 🎉🙌🏽🎉

Super stoked and thankful that you chose to put time and effort into this project!

**Comment by erichodges** [comment]
@chowdhurian Cool, thank you and thanks to everyone else. I'm happy to help if I can. I need to look through the open issues again.

**Pull request merged**
